2019-06-19 08:52:19: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 08:52:41: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:06:08: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:13:42: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:13:52: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:27:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:28:29: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:48:30: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:48:30: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:48:30: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:48:31: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:48:31: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:48:31: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:48:32: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:53:48: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 09:54:41: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:06:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:14: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:49: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 10:09:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 14:06:28: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 14:19:19: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 14:20:28: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 14:21:11: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 14:26:45: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 14:56:51: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:22:00: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:40: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:41: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:41: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:41: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:41: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-06-19 15:28:41: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:41: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:41: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-06-19 15:28:41: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
